caddyhttp/log/log.go

```diff
@@ -67,17 +67,19 @@ func (l Logger) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, error) {
 
 			// Write log entries
 			for _, e := range rule.Entries {
-
-				// Mask IP Address
-				if e.Log.IPMaskExists {
-					hostip, _, err := net.SplitHostPort(r.RemoteAddr)
-					if err == nil {
-						maskedIP := e.Log.MaskIP(hostip)
-						// Overwrite log value with Masked version
-						rep.Set("remote", maskedIP)
+				if len(e.Log.Exceptions) == 0 || e.Log.ShouldLog(r.URL.Path) {
```

Comment:
This is just a style preference, but instead of adding another level of indent, how about if we invert this statement to `len(e.Log.Exceptions) > 0 && !e.Log.ShouldLog(r.URL.Path) {` and then you can just `continue` instead of nesting all the logic.

Reply:
I think this is less clear, but I've made the change :)